tox.ini

```diff
@@ -1,25 +1,26 @@
 [tox]
-envlist = py{27,34,35,36}-django{111,2}
+envlist = py{35,36,37}-django{111,2,22}
 
 [testenv]
 commands = coverage run --parallel -m pytest {posargs}
 deps =
     -r{toxinidir}/requirements.txt
-    django-oscar>=1.6,<1.7
+    django-oscar>=2.0
     django111: django>=1.11,<1.12
     django2: django>=2,<2.1
```

Comment:
Django 2.0 and 2.1 are end of life, so let's drop them from the tox configuration.